cmd/static-server/main.go

```diff
@@ -80,17 +80,20 @@ func loadStaticNMState(env *env.EnvInputs, nmstateDir string, imageServer imageh
 			return errors.WithMessagef(err, "problem generating ignition %s", f.Name())
```

Comment:
can't this stuff (line 59 downwards be in the image provider?)

Reply:
There's quite a few k8s-isms in the ImageProvider API. Probably more trouble than it's worth to convert everything to that format.